### DISCUSSION TITLE
Could I use other python environment in autokey besides the system default?

### DISCUSSION THREAD

**Discussion by sim2pro** [autokey/autokey#741] [Q&A] [answered]
I have difficulty installing packages for system default python. So I want autokey to use other python env on my computer, for example pyenv virtualenv. Could I, how?

Thank you!

**Comment by josephj11** [comment, marked as answer]
That one is beyond me.
Although asking such questions here in our discussion section is perfectly fine, you are likely to get better support asking on Gitter. (See boiler plate below.)

----

BTW: Whenever you have a question (as opposed to an issue) it's better to post it on our [list](https://groups.google.com/forum/#!forum/autokey-users) or on [Gitter](https://gitter.im/autokey/autokey). The list is more user oriented whereas more technical questions will get a better answer on Gitter.

Both of these venues have a number of active users, so you are likely to get more, better, and faster answers on them than here on GitHub where only our very busy developers and a few users see the posts. We also prefer to have the developers developing rather than taking time to answer questions and manage issues that the user community could handle for them.
